code/datums/loadout/item_representation/_item_representation.dm

```diff
@@ -66,19 +66,52 @@
 	for(var/atom/thing_in_content AS in item_to_copy.contents)
 		if(!isitem(thing_in_content))
 			continue
-		if(!is_savable_in_loadout(thing_in_content))
```

Comment:
we are saving every item, and check if they are available when equipping the loadout